.github/workflows/npm-cd.yml

```diff
@@ -95,12 +141,20 @@ jobs:
                   always-auth: true
                   token: ${{ secrets.NPM_AUTH_TOKEN }}
             
+            - name: Setup node
+              if: ${{ matrix.build.TARGET == 'aarch64-unknown-linux-musl' }}
+              working-directory: ./node
+              run: |
+                npm config set registry https://registry.npmjs.org/
+                npm config set '//registry.npmjs.org/:_authToken' ${{ secrets.NPM_AUTH_TOKEN }}
```

Comment:
isn't the second config set is enough? since it already contains the registry endpoint

Reply:
will try

Reply:
Not possible - two different settings.
Setting the registry, which don't have auth option, and then setting the auth. Sometime google is know better than us  😝